esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -9,12 +9,12 @@
 [scheduling]
    cycling mode = integer
    initial cycle point = 1
-   final cycle point = 5
+   final cycle point = 10
     [[graph]]
-        R1 = """install_cold => configure => process<fast> => compare<fast>
-                configure => process<medium> => compare<medium>"""
-        3/P2 = compare<fast>[-P2] => configure => process<fast> => compare<fast>
-        4/P3 = compare<medium>[-P3] => configure => process<medium> => compare<medium>
+        R1 = install_env_file => clone_latest_esmval => configure
+        2/P3 = configure[-P1] => process<fast> => compare<fast>
+        3/P3 = configure[-P2] => process<medium> => compare<medium>
+        4/P3 = compare<fast>[-P2] & compare<medium>[-P1] => clone_latest_esmval => configure
```

Comment:
Should the `clone_latest_esmval` task be run at the start of each cycle? (which reminds me, the documentation will need updating to include this new task) :)

Reply:
I forgot that we discussed this and we agreed to create a `housekeeping` task to manage the deletion of old clones.

Reply:
All recipes should run on the first cycle:
```suggestion
        R1 = """install_env_file => clone_latest_esmval => configure => process<fast> => compare<fast>
                    configure =>  process<medium> => compare<medium>"""
        2/P3 = configure[-P1] => process<fast> => compare<fast>
        3/P3 = configure[-P2] => process<medium> => compare<medium>
        4/P3 = compare<fast>[-P2] & compare<medium>[-P1] => clone_latest_esmval => configure
```